CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ so any changes need to be compliant with that spec
 
 ## Slack
 
-Email [ian@nobl9.com](ian@nobl9.com) to get an invite to our slack channel
+Email [mateusz.hawrus@nobl9.com](mateusz.hawrus@nobl9.com) to get an invite to our slack channel
```

Comment:
Currently you are able to join slack via link on openslo.com site. You can add this info in here if you don't want to be spammed too much.

Reply:
yeah, that probably makes more sense :) thanks